.github/workflows/workflow.yaml

```diff
@@ -50,13 +50,30 @@ jobs:
           	EOF
           fi
 
+      - name: Run tests (bzlmod)
+        if: matrix.bazel_mode == 'module'
+        working-directory: ./tests
```

Comment:
Thanks for this, I've removed the `_running_workspace`. The separation was done because the bzlmod tests couldn't be run from the root and for workspace I went with the convention to call from the root.